src/packaging/docker-build/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2.1'
+version: '2.2'
```

Comment:
Because the `reaper-build-packages` service references the _src/packaging/docker-build/docker-compose.yml_ which is set to version 2.2. Is this change ok or should we drop the other Docker Compose file back to 2.1?

Reply:
I would have downgraded `src/packaging/docker-build/docker-compose.yml` to 2.1 and seen if that worked. But since there's another file that's cross-referenced, I'd assume it's already at its lowest version.

This comment was more cosmetic, but always good to try to keep the version dependency lower when/if possible.

Reply:
That is a fair call. I will downgrade _src/packaging/docker-build/docker-compose.yml_ and _src/packaging/docker-build/docker-compose.yml_ to version 2.1, then test to make sure it still works.